services/graph/pkg/identity/ldap_education_school.go

```diff
@@ -141,7 +153,7 @@ func (i *LDAP) CreateEducationSchool(ctx context.Context, school libregraph.Educ
 		logger.Debug().Err(err).Msg("error adding school")
 		if errors.As(err, &lerr) {
 			if lerr.ResultCode == ldap.LDAPResultEntryAlreadyExists {
-				err = errorcode.New(errorcode.NameAlreadyExists, lerr.Error())
+				err = errorcode.New(errorcode.NameAlreadyExists, "A school with that name is already present")
```

Comment:
Can we make this string a `var` and use it in all 4 places?

Reply:
It's two different error messages. But yeah, your still right ;-)

Reply:
Fixed. And also improved logging and error handling a bit.